toolchest_client/files/input_util.py

```diff
@@ -0,0 +1,57 @@
+"""
+toolchest_client.files.input_util
```

Comment:
I agree that this is a good point to separate this logic into it's own module.

The name (`input_util`) and logical grouping ("Utilities for handling input files/paths") doesn't make sense to me, though. What functions do we put in here, but not `general` or another module that pops up for input files or output files?

Would this fit in `general`, or could this be renamed?

Reply:
Done re: the unit test! It's in `toolchest_client/files/tests/test_input_params.py`.
I agree that the name isn't great. I'd like to refactor it, but I'm still trying to think of a logical name.

Reply:
I'm going to refactor this into `general.py` and move on.